### DISCUSSION TITLE
ccf-validation-tools/Placenta/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#281] [General]
# ccf-validation-tools/Placenta/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Placenta/comments/

**Comment by emquardokus** [comment]
Placenta-Full-term_v1.1_DRAFT
https://docs.google.com/spreadsheets/d/1JeeIxpcyD8eWAgdNuKtf1OMlTzEw_6AKrS5OVDkC7t4/edit#gid=231591207

**Terms not found**
Recently added, Ticket merged 3069, not in the release yet @aleixpuigb
This report provides a list of terms not found neither in UBERON nor in CL. Please remove these terms from the ASCT+B table - disconsider this message if a term was recently added to the ontology.
    UBERON:8600019
    UBERON:8600020
    UBERON:8600021
placental basal plate (UBERON:8600019)
chorionic plate blood vessel (UBERON:8600021)
placental septum (UBERON:8600020)

**Different labels**
FIXED In row 34, the term CL:2000091 has different name/label in the source ontology. The name/label in the ASCT+B table is endometrial microvascular endothelial cells and the one in the ontology is endometrial microvascular endothelial cell. For reference, the given name/label by SMEs is endometrial microvascular endothelial cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Blank ontology ID**
New Term Requests are pending for all @aleixpuigb
Placenta tab: https://docs.google.com/spreadsheets/d/1BihrOU9vWrsA_wYjcbdPUr42_Sb22CO3gkBDieiYQPs/edit#gid=924260064

**Reply by aleixpuigb** [reply]
With the new release the following terms have been added in uberon:
placental basal plate (UBERON:8600019)
placental septum (UBERON:8600020)
chorionic plate blood vessel (UBERON:8600021)

**Comment by aleixpuigb** [comment]
New terms added after the Uberon release (v2024-01-18) and CL release (v2024-02-13):

Placenta: placental villous stroma' (UBERON:8600023)
Placenta: intervillous space' (UBERON:8600025)
Placenta: chorionic villous tree' (UBERON:8600026)
Placenta: chorionic stem villus' (UBERON:8600027)
Placenta: chorionic mesenchymal villus' (UBERON:8600028)
Placenta: chorionic immature intermediate villus' (UBERON:8600029)
Placenta: chorionic terminal villus' (UBERON:8600030)
Placenta: chorionic mature intermediate villus' (UBERON:8600031)
Placenta: chorionic floating villus' (UBERON:8600032)
Placenta: chorionic anchoring villus' (UBERON:8600033)
Placenta: placental disc (UBERON:8600038)
Placenta: chorionic villous mesenchyme (UBERON:8600039)
Placenta: chorionic stem villous blood vessel (UBERON:8600040)
Placenta: chorionic terminal villous capillary (UBERON:8600041)
Placenta: endometrial lymphatic vessel (UBERON:8600042)
Placenta: interstitial extravillous trophoblast cell' (CL:4033062)
Placenta: endovascular extravillous trophoblast cell' (CL:4033063)
Placenta: uterine resident macrophage' (CL:4033064)

A suggested edition of the Placenta tables for the chorionic cilli can be found in [these slides](https://docs.google.com/presentation/d/1mXSakDlwEUvI1uEDYK0eLubfia17JIuV3leHexY9nsw/edit#slide=id.g2a3cf6cdb5d_0_17).

**Comment by anitacaron** [comment]
New term in the ontology:
Placenta : trophoblast island (UBERON:8600050)

**Comment by aleixpuigb** [comment]
**New terms CL v2025-01-08

Placenta : placental macrophage (CL:4033087)
Placenta : decidual macrophage (CL:4033088)
